invertedai/logs/debug_logger.py

```diff
@@ -0,0 +1,77 @@
+from pydantic import BaseModel, validate_arguments
+from typing import List, Optional, Dict, Tuple
+from datetime import datetime
+
+import json
+
+class DebugLogger:
+
+    def __init__(self,debug_log_path):
+        self.debug_log_path = debug_log_path
+
```

Comment:
It is possible for some endpoints to not be called in edge cases but since this is a debug log and will be mainly used for visual observation, I think this will be fine to not necessarily explicitly have the log contain these keys.

Reply:
Done.